code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -52,19 +52,23 @@
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | LAVA_PROOF
 	slowdown = 0
-	armor = list("melee" = 70, "bullet" = 40, "laser" = 10, "energy" = 10, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 70, "bullet" = 50, "laser" = 30, "energy" = 10, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
A lavaland suit shouldn't have 2x as much melee resist, >3x as much bullet resist, and the same laser/energy resist, as a Security Hardsuit from the armory.

If you want it to be really good against, say, brute damage, that's okay. But don't make it amazing at melee damage, good at brute damage, as good as a sec hardsuit against laser/energy damage, AND additionally make you totally immune to lava.

Reply:
Those are the same values used by the ash drake armor, except the drake armor has 30 in bullet and 50 in laser. I dont really see the issue with those being stronger than security hardsuit when the sec hardsuit is 1. spaceproof 2. a weak piece of armor to begin with.

Reply:
Much of the loot on lavaland is OP compared to the rest of the game, and arguably deserves nerfing. So, comparing this to another lavaland item is probably not going to convince me, especially when this has lavaproofing in addition to its value as armor.

Reply:
I feel like armor/clothes in general deserves a major balance pass PR, especially lavaland stuff. I'll think about it. For this PR, I suppose you're right.

Reply:
I could lower them if that is wanted, though i must say that i find it making little sense that the drake armor that is often gotten in a round is stronger and better at everything but bullet damage. Not only is the Drake armor stronger and more ofte gotten, there is nothing stopping a miner that has killed a lot of drakes to give out this armor to the crew,, as there is two armor set pr 2 dragons killed. I think it makes sense that the one set you get from killing the king of slaugther demons is both stronger considering it's unique and very very hard to get

Reply:
Then go ahead and nerf the drake armor to balance out how easy it is to get.

Reply:
I will will turn the the armor values on it back in that case, but keep the hand and feet covering part plus lava proof.

Will change this when i get time later